dallinger/frontend/templates/dashboard_monitor.html

```diff
@@ -88,10 +81,37 @@ <h5 class="card-title">{{ pane }}</h5>
                         </div>
                     {% endfor %}
                 </div>
+            </div>
+            <div class="collapse p-3 bg-light" id="config">
+                <h4>Config</h4>
+                <div class="custom-control custom-switch">
 
+                    <input type="checkbox" class="custom-control-input" id="auto_recruit"
+                            {% if configuration.auto_recruit %}
+                           checked
+                            {% endif %}
+                    >
+                    <label class="custom-control-label" for="auto_recruit">Auto recruit</label>
+                </div>
+                <pre style="max-height: 400px; overflow: scroll; white-space: break-spaces">
+                    {%- if config_str -%}
+                        {{ config_str }}
```

Comment:
How come you need to pipe in both `config_str` and `configuration? Can't you serialize `configuration` to JSON from within Jinja?

Reply:
Yes, that's possible, but not with nice indentation. This is why I pass:
```
config_str=json.dumps(config, indent=2).strip()
```

Reply:
Does it not work if you write 

```
json.dumps(config, indent=2).strip()
```
within Jinja?